share/github-backup-utils/ghe-backup-settings

```diff
@@ -45,6 +45,11 @@ if [ "$GHE_VERSION_MAJOR" -ge 2 ]; then
     fi
 fi
 
+if ghe-ssh "$host" -- "which ghe-export-ssl-ca-certificates"; then
```

Comment:
I've not tested this, but isn't this going to produce unnecessary output in the backup output?  We might want to direct all output to /dev/null to be sure.